addons/mass_mailing/static/src/js/mass_mailing_form.js

```diff
@@ -1,10 +1,107 @@
 /** @odoo-module **/
 
 import FormView from 'web.FormView';
+import FormController from 'web.FormController';
 import FormRenderer from 'web.FormRenderer';
+import { bus, _t } from 'web.core';
 import viewRegistry from 'web.view_registry';
 import config from 'web.config';
 
+const MassMailingFormController = FormController.extend({
+    /**
+     * @override
+     */
+    init() {
+        this._super(...arguments);
+        this._boundTriggerIframeUpdated = this._triggerIframeUpdated.bind(this);
```

Comment:
We usually name event-triggered functions "_onXXX" and put them in their own "// Handlers" section, separated from the "// Private" one.